CONTRIBUTING.md

```diff
@@ -43,14 +49,50 @@ Suppose for your PR you create a branch on `root.git`.
 Our CI infrastructure automatically picks up a branch with the same name in your fork of `roottest.git`
 and use that for testing your PR.
 
+Building ROOT with your changes
+-----------------------
 
-Continuous Integration
-----------------------
+The source code for ROOT is kept in [GitHub](https://github.com/root-project/root).
+In general, with the sources in `../src`, you build ROOT with `cmake -Droottest=On -Dtesting=On -Dall=On ../src && cmake --build`.
+If possible, configure with ninja, passing `-G Ninja` to the first `cmake` invocation.
```

Comment:
```suggestion
In general, with the sources in `../src`, you build ROOT with:
```
   cmake -Droottest=ON -Dtesting=ON -Dall=ON ../src && cmake --build
```
If possible, configure with ninja, passing `-G Ninja` to the first `cmake` invocation.
```

(github is confused by the inner "```"

Reply:
I don't understand this comment:
- "github is confused": where and how? How did you assess / how do you see this confusion?
- "the inner "\`\`\`"": which inner "\`\`\`"? Which outer "```"? The line you commented on only has \`
- "stray backslash": I see no backslash in the diff.